classes/order/Order.php

```diff
@@ -2282,13 +2282,8 @@ public function getProductTaxesDetails($limitToOrderDetails = false)
                 $order_discount_tax_excl -= $order_cart_rule['value_tax_excl'];
             }
         }
-
-        $products_tax    = $this->total_products_wt - $this->total_products;
-        $discounts_tax    = $this->total_discounts_tax_incl - $this->total_discounts_tax_excl;
-
-        // We add $free_shipping_tax because when there is free shipping, the tax that would
-        // be paid if there wasn't is included in $discounts_tax.
-        $expected_total_tax = $products_tax - $discounts_tax + $free_shipping_tax;
```

Comment:
@maximebiloe $expected_total_tax is the products_tax (displayed in the CreditSlip; block on the left) so we don't need to substract the discount_tax or to add the free_shipping_tax, we may add it to the function that calculate the total displayed in the total block of CreditSlip (block on the right).